docs/docs/deployment/dagster-plus/ci-cd/branch-deployments/change-tracking.md

```diff
@@ -12,11 +12,13 @@ Branch Deployments Change Tracking makes it easier for you and your team to iden
 
 ## How it works
 
-Branch Deployments compare asset definitions in the branch deployment against the asset definitions in the main deployment. The UI will then mark changed assets accordingly. For example, if the pull request associated with the branch deployment adds a new asset, the UI will display a label indicating the addition.
+Branch Deployments compare asset definitions in the branch deployment against the asset definitions in the branches base deployment. The UI will then mark changed assets accordingly. For example, if the pull request associated with the branch deployment adds a new asset, the UI will display a label indicating the addition.
 
 You can also apply filters to show only new and changed assets in the UI. This makes it easy to understand which assets will be impacted by the changes in the pull request associated with the branch deployment.
 
-**Note:** The default main deployment is `prod`. To configure a different deployment as the main deployment, [create a branch deployment using the dagster-cloud CLI](/deployment/dagster-plus/ci-cd/branch-deployments/using-branch-deployments-with-the-cli) and specify it using the optional `--base-deployment-name` parameter.
+The default base for branch deployments is `prod`. To configure a different deployment as the base, [create a branch deployment using the dagster-cloud CLI](/deployment/dagster-plus/ci-cd/branch-deployments/using-branch-deployments-with-the-cli) and specify it using the optional `--base-deployment-name` parameter.
+
+The default behavior is to compare the branch deployment against the **current** state of the base deployment. Depending on deployment and development cadence, the code deployed in the base deployment can get ahead of the branch causing changes to appear that are from the base getting ahead of the branch. To address this, the CLI option `--snapshot-base-condition` can be set to either `on-create` to snapshot the base only when the branch is first created or `on-update` to refresh the base snapshot anytime the branch is updated.
```

Comment:
moved existing thing out of `Note` since this other piece of information feels similar - open to suggestions for better way to structure this

Reply:
That makes sense to me -- in my suggestion below, I recommend formatting the new paragraph as a note, since it's about setting options that deviate from the default behavior.

Reply:
should i put both in the same note or two separate notes?